server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -301,8 +327,10 @@ public long balance(SortedMap<TServerInstance,TabletServerStatus> current, Set<K
       return minBalanceTime;
 
     Map<String,String> tableIdMap = t.tableIdMap();
+    long now = System.currentTimeMillis();
```

Comment:
It doesn't look like this is being used more than once? Why pull it out to a variable in that case?